__tests__/app.test.js

```diff
@@ -63,6 +63,89 @@ describe('/api/articles/:article_id', () => {
+                        article_id: 1,
+                        title: "Living in the shadow of a great man",
+                        topic: "mitch",
+                        author: "butter_bridge",
+                        body: "I find this existence challenging",
+                        created_at: expect.any(String),
+                        votes: 102
+                    });
+                    return db.query("SELECT * FROM articles WHERE article_id=$1", [
+                        article_id,
+                    ]);
+                })
+                .then(({ rows }) => {
+                    expect(rows[0]).toMatchObject({
+                        article_id: 1,
+                        title: "Living in the shadow of a great man",
+                        topic: "mitch",
+                        author: "butter_bridge",
+                        body: "I find this existence challenging",
+                        created_at: expect.anything(),
+                        votes: 102
+                    });
+                });
+        });
+        test('400: Invalid input', () => {
+            return request(app)
+                .patch('/api/articles/1')
+                .send({})
+                .expect(400)
+                .then(({ body }) => {
+                    expect(body).toEqual({ msg: "Invalid input" });
+                    return db.query("SELECT * FROM articles WHERE article_id=1");
+                })
+                .then(({ rows }) => {
+                    expect(rows[0]).toMatchObject({
+                        article_id: 1,
+                        title: "Living in the shadow of a great man",
+                        topic: "mitch",
+                        author: "butter_bridge",
+                        body: "I find this existence challenging",
+                        created_at: expect.anything(),
+                        votes: 100
+                    });
+                });
+        });
+        test('400: Invalid input', () => {
+            const votes = {
+                inc_votes: 'two',
+            };
+            return request(app)
+                .patch('/api/articles/1')
+                .send(votes)
+                .expect(400)
+                .then(({ body }) => {
+                    expect(body).toEqual({ msg: "Invalid input" });
+                    return db.query("SELECT * FROM articles WHERE article_id=1"
+                    );
+                })
+                .then(({ rows }) => {
+                    expect(rows[0]).toMatchObject({
+                        article_id: 1,
+                        title: "Living in the shadow of a great man",
+                        topic: "mitch",
+                        author: "butter_bridge",
+                        body: "I find this existence challenging",
+                        created_at: expect.anything(),
+                        votes: 100
+                    });
+                });
+        });
```

Comment:
You need to add a test for if you try to update an article with:

- invalid id - e.g. '/api/articles/invalid'
- not found id - e.g. '/api/articles/9999999999'